akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -314,6 +314,7 @@ Akka Typed APIs are still marked as [may change](../common/may-change.md) and th
   prefer `Behaviors.withTimers`.
 * `TimerScheduler.startPeriodicTimer`, replaced by `startTimerWithFixedDelay` or `startTimerAtFixedRate`
 * `Routers.pool` now take a factory function rather than a `Behavior` to protect against accidentally sharing same behavior instance and state across routees.
+* Java @apidoc[EntityRef] ask timeout now takes a `java.time.Duration` rather than a @apidoc[Timeout]
```

Comment:
does it understand how to create such apidoc links when we have two classes?
maybe make it full class name anyway since the change is only in javadsl

Reply:
It found the scaladsl one, i'll be specific

Reply:
if there's only one match (like here, just the scaladsl one) it will point to that one.

Reply:
There's two as this one is the java api, will fix